giskard/llm/injection_data/check_injection_prompts_data.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
```

Comment:
??

Reply:
I added this script to be factored out when we move the dataset outside the repo

Reply:
yes, to impose some minimal requirements: name of columns, no null values.